tests/pipelines/stable_diffusion/test_stable_diffusion_panorama.py

```diff
@@ -260,6 +294,36 @@ def test_stable_diffusion_panorama_default(self):
 
         assert np.abs(expected_slice - image_slice).max() < 1e-2
 
+    def test_stable_diffusion_panorama_circular_padding(self):
```

Comment:
I think it's okay to remove this slow test.

Reply:
Removed